website/content/docs/release-notes/nomad/v1_8_x.mdx

```diff
@@ -0,0 +1,39 @@
+---
+layout: docs
+page_title: 1.8.x
+description: >-
+  Nomad release notes for version 1.8.x
+---
+
+# Nomad 1.8.0
+
+We are pleased to announce the following Nomad updates.
+
+## Release highlights
+
+- **Long Term Support Release (Enterprise):** Nomad 1.8.0 now introduces a long term supported release which provides extended maintenance (two years of critical fixes provided through minor releases and efficient upgrades (support for direct upgrades from one LTS release to the next). 
+
+- **Exec2 Task Driver (beta):** The `exec2` driver is used to execute a command for a task. It offers a security model optimized for running 'ordinary' processes with very low startup times and minimal overhead in terms of CPU, disk, and memory utilization. The `exec2` driver leverages kernel features such as the Landlock LSM, cgroups v2, and the unshare system utility. Due these enhancements, tasks no longer require leveraging filesystem isolation based on chroot, which provide an enhanced security profile and improved performance for the Nomad operator. Refer to [Exec2 Task Driver](/nomad/plugins/v1.8.x/drivers/exec2) for more details. 
+
+
+
+## What's deprecated
+
+- **LXC Task driver:** The [Community LXC Task Driver](/nomad/plugins/drivers/community/lxc) is now deprecated and will be removed in Nomad 1.9. Users are encouraged to migrate their workloads to another task driver such as the built-in Docker task driver[/nomad/docs/drivers/docker]. 
+- **ECS Task driver:** The remote [ECS Task Driver](https://developer.hashicorp.com/nomad/plugins/drivers/remote/ecs) is now deprecated and will be removed in Nomad 1.9. Users are encouraged to migrate their workloads to Amazon ECS or to another Nomad task driver. 
```

Comment:
Are we _removing_ these or just archiving the repos? Removing the repos entirely seems like kind of a rude move.

Reply:
I will say the repos will archived and support will be removed. Does that work?

Reply:
Sounds good!